core/server/web/api/v2/admin/middleware.js

```diff
@@ -53,17 +53,3 @@ module.exports.authAdminApi = [
     shared.middlewares.prettyUrls,
     notImplemented
 ];
-
-/**
- * Authentication for client endpoints
- */
-module.exports.authenticateClient = function authenticateClient(client) {
```

Comment:
At first thought, this bit was still used by unmerged schedules controller changes but looks like we have 2 different `authenticateClient` methods in routes for v2. A good reminder to remove the other one once schedules stuff goes in today

Reply:
The one in scheduler controller is I guess [this](https://github.com/TryGhost/Ghost/blob/master/core/server/services/auth/authenticate.js#L11) one, which is shared and used on v0.1, so I guess that should only be removed with v0.1 later? 🤔